src/pybind/mgr/snap_schedule/fs/schedule_client.py

```diff
@@ -293,6 +293,13 @@ def create_scheduled_snapshot(self,
                     log.info(f'created scheduled snapshot of {path}')
                     log.debug(f'created scheduled snapshot {snap_name}')
                     sched.update_last(time, db)
+                except cephfs.ObjectNotFound:
+                    # maybe path is missing or wrong
+                    self._log_exception('create_scheduled_snapshot')
+                    log.debug(f'path {path} is probably missing or wrong; '
+                              'remember to strip off the mount point path '
+                              'prefix to provide the correct path')
+                    sched.set_inactive(db)
```

Comment:
nit:maybe do this in a `finally` block?

Reply:
moved schedule deactivation to `finally` clause